automation/carbonite-position-check.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+VAULT_TOKEN=${1:-$(cat $HOME/.vault-token)}
+WORKING_DIR=$PWD
+
+JSON_CREDS=`docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN -e VAULT_ADDR=https://clotho.broadinstitute.org:8200 broadinstitute/dsde-toolbox vault read -format=json secret/dsde/firecloud/dev/common/firecloud-account.json | jq '.data'`
```

Comment:
hardcoding the vault hostname and port seems fraught (and apparently unnecessary?)

Reply:
This is copied from `submission-perf-test.sh` and I am not aware of a location from which to retrieve this address programmatically (definitely not from vault!)

Reply:
the command runs fine for me without it...